### DISCUSSION TITLE
WinUI 3 MVVM Documentation

### DISCUSSION THREAD

**Discussion by ward-horsfall** [microsoft/microsoft-ui-xaml#7819] [Q&A]
Hi,

Where is the specific reference for the WinUI 3 MVVM documentation?

Can somebody please supply a URL to me?

Thanks
Ward

**Comment by mohummedibrahim** [comment]
If you're looking for the docs for CommunityToolkit.MVVM package (which is platform-agnostic and supports .Net Standard 2.0), then here it's.
https://learn.microsoft.com/en-us/dotnet/communitytoolkit/mvvm/
https://learn.microsoft.com/en-us/windows/communitytoolkit/mvvm/introduction
